### ISSUE TITLE
Track javascript errors

### ISSUE THREAD

**Issue by artur-beljajev** [internetee/registry#429]
Saving javascript errors would allow us discover front-end bugs before a user notices them.

**Comment by teadur** [comment]
How would we track them ? from what i understand javascript is rendered in user browser ?

**Comment by artur-beljajev** [comment]
Yes, it is client-side. http://stackoverflow.com/questions/951791/javascript-global-error-handling

**Comment by teadur** [comment]
#208 and #211 are related.

**Comment by teadur** [comment]
@artur-beljajev so exactly how can we start using errbits js part.
From what i can understand from the docs we need to add the javascript lib and read variables from configuration. https://github.com/errbit/errbit#javascript-error-notifications

**Comment by artur-beljajev** [comment]
@teadur We decided it might be good idea to setup additional dedicated instance meant exclusively for front-end errors, so that the original instance remains unexposed to Internet.

What do you think?

Some work has been started in `track-js-errors` branch.
